parallel-install/pkg/deployment/coredns.go

```diff
@@ -0,0 +1,152 @@
+package deployment
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"html/template"
+	"strings"
+
+	"github.com/avast/retry-go"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/logger"
+	v1 "k8s.io/api/core/v1"
+	apierr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+var coreDNSPatchTemplate = `
+.:53 {
+    errors
+    health
+    rewrite name regex {{ .DomainName}} istio-ingressgateway.istio-system.svc.cluster.local
+    ready
+    kubernetes cluster.local in-addr.arpa ip6.arpa {
+      pods insecure
+      fallthrough in-addr.arpa ip6.arpa
+    }
+    hosts /etc/coredns/NodeHosts {
+      reload 1s
+      fallthrough
+    }
+    prometheus :9153
+    forward . /etc/resolv.conf
+    cache 30
+    loop
+    reload
+    loadbalance
+}
+`
+
+// CoreDNSPatch contains values to fill template with
+type CoreDNSPatch struct {
```

Comment:
Does this type has to be exported? From what I can see you use it only for internal purpose and do not expose any patching strategy to the library users.

Reply:
Done